src/components/cluster/Cluster.tsx

```diff
@@ -222,7 +222,13 @@ function ClusterForm({ id, cluster_name, server_url, active, config, environment
         {
             cluster_name: {
                 required: true,
-                validator: { error: 'Name is required', regex: /^.*$/ }
+                validators: [
+                                { error: 'Name is required',  regex: /^.*$/ },
+                                { error: "Use only lowercase alphanumeric characters, '-' or '.'", regex: /^[a-z0-9_\.]+$/ },
+                                { error: "Start/End with an alphanumeric character", regex: /^[a-z0-9](.*[a-z0-9])?$/ },
+                                { error: "Do not use spaces, '-' or '.'", regex: /^\S+$/ },
```

Comment:
it only checks for space, but this ll be caught by 227

Reply:
correct